openpype/plugins/publish/extract_review.py

```diff
@@ -77,6 +77,13 @@ class ExtractReview(pyblish.api.InstancePlugin):
     profiles = None
 
     def process(self, instance):
+
+        # Instance should be integrated on a farm
+        if instance.data.get("farm"):
```

Comment:
Oh never mind my prior comment haha I see you ended up doing exactly the same I did. Can we now then remove the plugins that delete the "review" family and change the logic behind the families transfer on submission to the farm?